.rubocop.yml

```diff
@@ -57,8 +57,14 @@ Security/JSONLoad:
 Style/Alias:
   EnforcedStyle: prefer_alias
 
-Style/BracesAroundHashParameters:
-  Enabled: false
+Style/HashEachMethods:
```

Comment:
Could you move unrelated change to another PR and elaborate why it's important to add? Thanks.

Reply:
I've opened the PR #61 for it.